core/tests/test_utils.py

```diff
@@ -839,6 +840,29 @@ def assertRaisesRegexp(  # pylint: disable=keyword-arg-before-vararg
             expected_exception, expected_regexp,
             callable_obj=callable_obj, *args, **kwargs)
 
+    def assert_matches_regexps(self, regexps, items):
+        """Asserts each item in the list matches the corresponding regexp."""
```

Comment:
Please add args, raises -- it's a bit unclear what exactly this is checking from the existing description alone, since you also have checks on lengths.

Reply:
Done.